Search_places.py

```diff
@@ -0,0 +1,30 @@
+from sqlalchemy import create_engine
+from mysql_details import *
+import pandas as pd
+
+#Fetch Data from Database
+def get_data():

```

Comment:
Fetching the whole table would be inefficient - can you instead craft a query which takes care of the common name matching and returns only the matching rows?

Reply:
Sure, I'll do that.